dev/com.ibm.ws.org.hibernate.validator.cdi/.classpath

```diff
@@ -3,5 +3,6 @@
 	<classpathentry kind="src" path="src"/>
 	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="src" path="/com.ibm.ws.kernel.service"/>
```

Comment:
Do we need this change?

Reply:
Good catch.  That was added by eclipse and shouldn't have been checked in.  I'll remove it